installation/routines/customize_options.sh

```diff
@@ -142,6 +142,58 @@ Would you like to overwrite your configuration? [Y/n]"
     fi
 }
 
+_option_spotify() {
+  # ENABLE_SPOTIFY
+  clear_c
+  print_c "---------------------- SPOTIFY ----------------------
+
+Installs an additional player, so that you can
```

Comment:
I am adding this comment without having reviewed the actually player logic. We should not have different players. If a user does not install Spotify, we should still install the same player which only uses MPD. 

With this logic, we would not install "an additional player" and instead, "just Spotify". But In understand that there is more to do than just providing credentials (e.g. Developer Account)

> Install and configure Spoify for your Phoniebox

Reply:
I don't really get what you mean.

The logic is:
MPD will always be installed.
The user can choose, if he wants to have an additional player (here Spotify). If he chooses "no" he will still have MPD